h2/settings.py

```diff
@@ -254,6 +254,13 @@ def __len__(self):
         return len(self._settings)
 
 
+    def __eq__(self,other):
+        return self._settings==other._settings
+
+    def __ne__(self,other):
+        return not self._settings==other._settings
```

Comment:
So it's usually best to implement this in terms of equality unless there's a good reason not to, so I'd write this as `return not self == other`.